### ISSUE TITLE
Open a specific Preference tab

### ISSUE THREAD

**Issue by chadwhitaker** [HackPlan/quark-shell-mac#49]
Perhaps you can pass in the tab identifier on `quark.openPreferences()` to open a specific tab?

Example: `quark.openPreferences('account')` would open the tab where `"identifier": "account"`.

**Comment by xhacker** [comment]
Good idea, however I’m a little busy these days. I’ll keep this as a feature request :)
